docs/INSTALL-GUIDE-LINUX.md

```diff
@@ -0,0 +1,80 @@
+# Linux support
+
+This tutorial uses Ubuntu 20.04.
+
+Unless specified otherwise, all linux distributions are supported.
+
+## 1. install node.js and npm
+
+It is recommend to use [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm#install--update-script),
+ use this link to install latest nvm version.
+
+For example:
+
+```Shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Then update your shell environment (for example with Ubuntu 20.04 in this example):
+
+```Shell
+source ~/.profile
+```
+
+Display available versions to install:
+
+```Shell
+nvm ls-remote
+```
+
+Install the node version you want, for example to install node 14.2.0.0:
+
+```Shell
+nvm install 14.2.0
+```
+
+see for details [the full tutorial's linux node install page](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3A1-linux-download-install-nodejs.md).
+
+## 2. install gtp2ogs using npm
+
+Install gtp2ogs locally, for example at the root of you user account,
+ create a new folder `gtp2ogs-node`.
+
+```Shell
+mkdir ~/gtp2ogs-node
+```
+
+Then go inside this folder and install gtp2ogs with npm:
+
+```Shell
+cd ~/gtp2ogs-node
+npm install gtp2ogs
+```
+
+gtp2ogs will be installed in `~/gtp2ogs-node/node_modules/gtp2ogs`
+
+## 3. Recommended: Upgrade to devel branch
+
+This step is **highly recommended**.
+
+By default, npm installs an old branch of gtp2ogs that does not include latest
+ improvements, new features, and fixes.
+
+You can go to `~/gtp2ogs-node/node_modules/gtp2ogs`, delete all existing
+ files (make a copy backup just in case), then either:
+
+- download ZIP from github [the devel branch](https://github.com/online-go/gtp2ogs/tree/devel),
```

Comment:
why not git clone?

Reply:
it is possible to use git (as explained below), but for beginners they may find it easier to copy paste the files directly

Reply:
but indeed i forgot to add the git clone command line, added :+1: